openedx/adg/lms/webinars/helpers.py

```diff
@@ -0,0 +1,55 @@
+"""
+Helpers for webinars app
+"""
+from django.apps import apps
+
+from openedx.adg.common.lib.mandrill_client.client import MandrillClient
+from openedx.adg.common.lib.mandrill_client.tasks import task_send_mandrill_email
+
+
+def _send_webinar_cancellation_emails(webinar_title, webinar_description, webinar_start_time, recipient_emails):
```

Comment:
We can have only a webinar param instead of 4 params. WDYT?

Reply:
Webinar is quite a big model. And we do not require any extra data (other than the data asked here) for the email. 
Expecting a couple of strings is better than expecting a complete object.